docs/source/sdk_chaincode.md

```diff
@@ -0,0 +1,53 @@
+Install Fabric Application SDKs
+-----------------------------------
+
+**Audience:** Architects, application and smart contract developers.  
+
+Hyperledger Fabric offers a number of SDKs to support developing
+applications in various programming languages. SDKs are available for
+Node.js and Java.  Please download the SDK of your language of choice.
+
+> -   [Node.js SDK](https://github.com/hyperledger/fabric-sdk-node) and
+>     [Node.js SDK
+>     documentation](https://hyperledger.github.io/fabric-sdk-node/).
+> -   [Java SDK](https://github.com/hyperledger/fabric-gateway-java) and
+>     [Java SDK
+>     documentation](https://hyperledger.github.io/fabric-gateway-java/).
+>
+> Prerequisites for developing with the SDKs can be found in the Node.js
+> SDK
+> [README](https://github.com/hyperledger/fabric-sdk-node#build-and-test)
+> and Java SDK
+> [README](https://github.com/hyperledger/fabric-gateway-java/blob/master/README.md).
+
+In addition, there are two more application SDKs that have not yet been
+officially released (for Python and Go), but they are still available
+for downloading and testing:
+
+> -   [Python SDK](https://github.com/hyperledger/fabric-sdk-py).
+> -   [Go SDK](https://github.com/hyperledger/fabric-sdk-go).
+
```

Comment:
The Go SDK is now v1.0.0 so I think it can be moved to the list above. Would this be the readme?
https://github.com/hyperledger/fabric-sdk-go/tree/v1.0.0

Reply:
I think the link for docs would be:
https://pkg.go.dev/github.com/hyperledger/fabric-sdk-go

Reply:
Done.